### ISSUE TITLE
[TW-1910] unreachable statement

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#1923]
_Martin Strunz on 2017-05-08T14:03:37Z says:_ 

 \src\Variant.cpp line 1965
unreachable
return false;

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2017-05-08T14:03:37Z
Modified: 2017-05-08T15:23:45Z
```

**Comment by taskwarrior** [comment]
_Tomas Babej on 2017-05-08T15:23:13Z says:_ 

 Thanks! Fixed in 2.6.0.
